source/blogs/dbc/dbc.rst

```diff
@@ -42,105 +42,121 @@ is a Debug Host and the other is a Debug target. This is achieved through emulat
 a debug device by using xDBC on the debug target. The debug device presented by the
 debug target can be used by debug host for low level system debugging of target.
 
-Github PR's link for enabling adb over DbC for Celadon KBL NUC
---------------------------------------------------------------
+Github PRs for enabling adb over DbC for |C| KBL NUC
+----------------------------------------------------
 
-1) Kernel and adb changes:
-   https://github.com/projectceladon/vendor-intel-utils/pull/291
-2) Adding android property to switch between adb over DWC and DBC: 
-   https://github.com/projectceladon/device-androidia-mixins/pull/274
-3) Adding sepolicy to access DbC RAW device node:
-   https://github.com/projectceladon/device-androidia/pull/416
+#. `Kernel and adb changes <https://github.com/projectceladon/vendor-intel-utils/pull/291>`_
+#. `Adding android property to switch between adb over DWC and DBC <https://github.com/projectceladon/device-androidia-mixins/pull/274>`_
+#. `Adding sepolicy to access DbC RAW device node <https://github.com/projectceladon/device-androidia/pull/416>`_
 
 Steps to enable ADB over DbC support in Host Machine
 ----------------------------------------------------
 
-1. Need to edit below file, if there is no such file then create it. Make sure you
-   have root access to write to this file.
-   
-   .. image:: images/edit_udev.png
+#. Add the following permission to the udev rule */etc/udev/rules.d/51-android.rules*. Create the file if it doesn't exist.
```

Comment:
Suggested rewrite:
...Add the following permission to the udev rule */etc/udev/rules.d/51-android.rules* (create the file if it does not exist):